SpatialGDK/Source/SpatialGDK/Private/Schema/ActorOwnership.cpp

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "Schema/ActorOwnership.h"
+
+#include "Engine/NetConnection.h"
+#include "GameFramework/Actor.h"
+#include "GameFramework/PlayerController.h"
+
+#include "EngineClasses/SpatialPackageMapClient.h"
+
+namespace SpatialGDK
+{
+ActorOwnership ActorOwnership::CreateFromActor(const AActor& Actor, const USpatialPackageMapClient& PackageMap)
+{
+	ActorOwnership Ownership;
+	UNetConnection* OwningConnection = Actor.GetNetConnection();
+	if (IsValid(OwningConnection))
+	{
+		const Worker_EntityId ControllerEntity = PackageMap.GetEntityIdFromObject(OwningConnection->PlayerController);
+		check(ControllerEntity != SpatialConstants::INVALID_ENTITY_ID);
+		Ownership.OwnerActorEntityId = ControllerEntity;
+	}
+	else
+	{
+		Ownership.OwnerActorEntityId = SpatialConstants::INVALID_ENTITY_ID;
```

Comment:
Is this expected or unexpected? Is it worth logging it as a warning or error?

Reply:
No, this is normal. Do you think a comment would be nice?